src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+function formatCells(partOfTable) {
+  const rows = [...document.querySelector(partOfTable).rows];
+
+  if (rows.length <= 0) {
+    return;
+  }
+
+  for (const row of rows) {
+    const clonedCell = row.cells[1].cloneNode(true);
```

Comment:
Cloning the second cell of each row is correct, but ensure that each row has at least two cells to avoid errors. Consider adding a check to ensure `row.cells[1]` exists before attempting to clone it.